timesync

```diff
@@ -99,7 +103,27 @@ def parse_all_args():
     return parser.parse_args()
 
 
+def read_config():
+    """Returns a dictionary representing the contents of the json file
+    $HOME/timesyncconf.json or a set of sensible defaults if that file does
```

Comment:
Doesn't it return a sensible list of defaults no matter what, even if those are overwritten by the config?